ce/artifacts/SetOfDemands.ts

```diff
@@ -24,6 +24,13 @@ export class SetOfDemands {
     }
   }
 
+  /** Async Initializer */
```

Comment:
Nit: might rename file to "setOfDemands" to stay consistent with lowercase beginnings of names.